js/script.js

```diff
@@ -1,16 +1,13 @@
 function buildColorDropdown(theme) {
-    // cornflowerblue - Cornflower Blue (JS Puns shirt only
-    // darkslategrey - Dark Slate Grey (JS Puns shirt only)
-    // gold - Gold (JS Puns shirt only)
-    // tomato - Tomato (I &#9829; JS shirt only
-    // steelblue - Steel Blue (I &#9829; JS shirt only)
-    // dimgrey - Dim Grey (I &#9829; JS shirt only)
-    var colors = null;
+    var colors = null
+    var descriptions = null
```

Comment:
Let's remember to talk about how we could use javascript objects here

Reply:
Great, looking forward to it!